### ISSUE TITLE
AutoML release request

### ISSUE THREAD

**Issue by aravindvnair99** [tensorflow/tfjs#5560]
We can release a new version with the latest tfjs dependency if that’s what you need. Please keep an eye in the next few weeks.

_Originally posted by @lina128 in https://github.com/tensorflow/tfjs/issues/5256#issuecomment-886082504_

AutoML is still stuck with old and outdated dependencies which causes issues when using npm 7 due to peer dependencies. I request it be updated a new release be made alongside 3.9.0 released 11 hours ago.

**Comment by aravindvnair99** [comment]
Hey @mattsoulanille!

I was wondering if you had any updates on this... Shall I give you a PR?

**Comment by rthadur** [comment]
This is done , related PR has been merged here https://github.com/tensorflow/tfjs/pull/5570 , thank you @mattsoulanille @aravindvnair99

**Comment by aravindvnair99** [comment]
> This is done , related PR has been merged here #5570 , thank you @mattsoulanille @aravindvnair99

Thank you @rthadur and @mattsoulanille! But no release has been made yet for `@tensorflow/tfjs-automl`:

- https://github.com/tensorflow/tfjs/tags
- https://www.npmjs.com/package/@tensorflow/tfjs-automl

The published version is still version `1.2.0` from 4 months ago. Could a release be made for recent commits?

**Comment by mattsoulanille** [comment]
@aravindvnair99 Sorry to keep you waiting on this. We plan to release automl with the latest changes from that PR in the near future. I'll update this issue when it's released.

**Comment by mattsoulanille** [comment]
Hi @aravindvnair99.

Automl has just now been released. Sorry for the wait. We're working on improving our release automation so we can have more consistent releases, but automl was not part of the automation yet. It has now been added in #5615, so you can expect more consistent releases in the future.

**Comment by aravindvnair99** [comment]
> Hi @aravindvnair99.
> 
> Automl has just now been released. Sorry for the wait. We're working on improving our release automation so we can have more consistent releases, but automl was not part of the automation yet. It has now been added in #5615, so you can expect more consistent releases in the future.

Thanks a lot for the release and quick reply @mattsoulanille! Closing this issue tracker for now.
